src/infrastructure/jira/jira-client/jwt-utils.ts

```diff
@@ -9,7 +9,7 @@ export type JwtTokenParams = {
 		readonly query?: Record<string, unknown>;
 		readonly body?: Record<string, unknown>;
 	};
-	readonly connectClientKey: string;
+	readonly connectAppKey: string;
```

Comment:
From the Connect [docs](https://developer.atlassian.com/cloud/bitbucket/understanding-jwt-for-apps/#claims) (emphasis mine):

> The issuer of the claim. Connect uses it to identify the application making the call. For example:
> **If the Atlassian product is the calling application: contains the unique identifier of the tenant. This is the clientKey** that you receive in the installed callback. You should reject unrecognized issuers.
> **If the app is the calling application: the app key specified in the app descriptor**.

We were correctly using the app key to generate JWTs, but the variable was mislabelled in this method.